qmd/_dictionary-version.qmd

```diff
@@ -1,9 +1,14 @@
 <!--Source constants file until I get clarity for environment usage across entire package or if sourcing is enough -->
 `r source("R/constants.R")`
 ```{r, echo=FALSE}
-dictionary_full_file_name <- list.files(file.path(getwd(),constants$dictionary_directory), pattern = "^ODM.*xlsx$")
-dictionary_version_pattern <- "ODM_dictionary_(.*?).xlsx"
+source("R/version-utils.R")
+
+dictionary_full_file_names <- list.files(file.path(getwd(),constants$dictionary_directory), pattern = "^ODM.*xlsx$")
```

Comment:
Why are we using two patterns? This one and the one in the `get_latest_version` function?